packages/dnb-eufemia/src/extensions/forms/FieldBlock/FieldBlock.tsx

```diff
@@ -141,6 +141,8 @@ export type Props<Value = unknown> = SharedFieldBlockProps &
     disableStatusSummary?: boolean
     /** For internal use only */
     required?: boolean
+    /** Role for the fieldset element when using fieldset */
+    fieldsetRole?: string
```

Comment:
```suggestion
    fieldsetRole?: 'group' | 'radiogroup'
```

Technically fieldset can also have role `"presentation"` but I don't think that's needed here.